ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -425,11 +436,8 @@ impl<'a> TraceBuilder<'a> {
                 let load = jit_ir::LookupGlobalInst::new(self.handle_global(*gd_idx)?)?;
                 self.jit_mod.push_and_make_operand(load.into())
             }
-            aot_ir::Operand::Arg { arg_idx, .. } => {
-                // Lookup the JIT instruction that was passed into this function as an argument.
-                // Unwrap is safe since an `Arg` means we are currently inlining a function.
-                // FIXME: Is the above correct? What about args in the control point frame?
-                Ok(self.frames.last().unwrap().args[usize::from(*arg_idx)].clone())
+            aot_ir::Operand::Arg { .. } => {
```

Comment:
So we don't completely remove `Operand::Arg`?

Reply:
Yes, I realised that too now. Working on it.

Reply:
Try [563e094](https://github.com/ykjit/yk/pull/1261/commits/563e0942d0190c57c947ceca6931e6dc7588a4a6)